### PR TITLE
Added AIRLink flight controller page

### PR DESCRIPTION
**Pull request by aviaks**
AIRLink added to the list of supported flight controllers.

### CODE REVIEW

File: en/flight_controller/airlink.md [hunk 1]
```diff
@@ -0,0 +1,1003 @@
...
+:::
+
+AIRLink has two computers and integrated LTE Module: 
+
+   - The flight control computer (autopilot) has a triple-redundant vibration-dampened and temperature-stabilized IMU. 
+   - The powerful AI mission computer enables advanced drone software features like computer vision and obstacle avoidance, digital HD video streaming, and payload data streaming.
+   - LTE/5G and WiFi connectivity modules provide permanent broadband internet connection which is enabler for remote workflows.
+
+
+![Three in one](../../assets/flight_controller/airlink/airlink-3in1.png)
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:23]: I would remove this image, or move it. Fine on a marketing doc, but just a waste of information space here - says nothing you didn't have in your list above.

File: en/flight_controller/airlink.md [hunk 2]
```diff
@@ -0,0 +1,1003 @@
...
+
+## Feature highlights
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:25]: We use first letter capitalization

```suggestion
## Feature Highlights
```

File: en/flight_controller/airlink.md [hunk 3]
```diff
@@ -0,0 +1,1003 @@
...
+   - Up to 600 Mbps bandwidth
+   - 5G sub-6 and mmWave, SA and NSA operations
+   - 4G Cat 20, up to 7xCA, 256-QAM DL/UL, 2xCA UL
+   - 4 x 4 MIMO for 4G and 5G (sub-6 bands)
+   - 3G HSPA+
+   - Approved by JRL/JTBL, FCC, PTCRB, RED, GCF
+   - Antenna, 4x4 MIMO
+   - Bands: Worldwide
+
+## Set content
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:81]: Would be good to improve this title - it sounds like an instruction, but I think you mean "this is what is you get when you buy it"?

Is this the "Airlink Enterprise", or do you get the same thing in both?

File: en/flight_controller/airlink.md [hunk 4]
```diff
@@ -0,0 +1,1003 @@
...
+   <td>198 g</td>
+   <td>89 g</td>
+   </tr>
+   <tr>
+   <td>Ambient temperature</td>
+   <td>-40°C-..+50°C</td>
+   <td>-40°C-..+50°C</td>
+   </tr>
+   </tbody>
+   </table>
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:140]: Easier to maintain in markdown for this width table. I used https://tableconvert.com/html-to-markdown with simple markdown table setting

```suggestion
Parameter  | AIRLink Enterprise  | AIRLink Core  
--- | --- | ---
Enclosure | Aluminum, with integrated heatsink and fan mounting option. | External heatsink or reasonable power dissipation should be provided by the design. 
Dimensions | L103 x W61 x H37 mm | L100 x W57 x H22 mm 
Weight | 198 g | 89 g 
Ambient temperature | -40°C-..+50°C | -40°C-..+50°C 
```

Comment (hamishwillee) [reply en/flight_controller/airlink.md:140]: Ditto for other tables.

File: en/flight_controller/airlink.md [hunk 5]
```diff
@@ -0,0 +1,1003 @@
...
+
+## Key features
+
+- **Easy to mount**
+
+![Easy mount](../../assets/flight_controller/airlink/airlink-easy-to-mount.jpg)
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:146]: Aligns image with bullet.

```suggestion
  ![Easy mount](../../assets/flight_controller/airlink/airlink-easy-to-mount.jpg)
```

File: en/flight_controller/airlink.md [hunk 6]
```diff
@@ -0,0 +1,1003 @@
...
+
+- **FPV camera comes as standard**
+
+![FPV camera comes as standard](../../assets/flight_controller/airlink/airlink-fpv-camera.jpg)
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:150]:
```suggestion
  ![FPV camera comes as standard](../../assets/flight_controller/airlink/airlink-fpv-camera.jpg)
```

File: en/flight_controller/airlink.md [hunk 7]
```diff
@@ -0,0 +1,1003 @@
...
+
+Frequently asked questions are answered in [FAQ](https://docs.sky-drones.com/airlink/faq)
+
+
+## Where to Buy
+
+Purchase from the original Sky-Drones Store (worldwide shipping with 1-2 days order processing time): 
+
+ - [Buy AIRLink 4G](https://sky-drones.com/sets/airlink-enterprise-set.html)
+ - [Buy AIRLink 5G](https://sky-drones.com/sets/airlink-5g-enterprise-set.html)
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:994]: - Should these indicate edition? E.g. Airlink 4G Enterprise?
- Should there be a purchase link for the "Core"?

File: en/flight_controller/airlink.md [hunk 8]
```diff
@@ -0,0 +1,1003 @@
...
+For more information and instructions on setting up and using the AIRLink system see  [AIRLink Documentation](https://docs.sky-drones.com/airlink/)
+
+For technical help, support and customization please get in touch at [Sky-Drones contact page](https://sky-drones.com/contact-us)
+
+More information can be found at  [www.sky-drones.com](https://sky-drones.com)
+
+Frequently asked questions are answered in [FAQ](https://docs.sky-drones.com/airlink/faq)
+
+
+## Where to Buy
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:989]: Consider moving this up near the top of the doc - it doesn't take up much space and often users are just trying to find the right link for the version to use with PX4.

File: en/flight_controller/airlink.md [hunk 9]
```diff
@@ -0,0 +1,1003 @@
...
+:::
+
+To [build PX4](../dev_setup/building_px4.md) for this target:
+```
+make sky-drones_smartap-airlink
+```
+
+## Peripherals
+
+Peripherals information coming soon.
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:966]: FMI, ETA on that?

File: en/flight_controller/airlink.md [hunk 10]
```diff
@@ -0,0 +1,1003 @@
...
+   </tbody>
+   </table>
+
+DShot capabiltiy can be used on the first four AUX pins.
+
+## Building firmware
+
+:::tip
+Most users will not need to build this firmware!
+It is pre-built and automatically installed by *QGroundControl* when appropriate hardware is connected.
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:956]: What versions of PX4 can this work with? Just main, or earlier?

File: en/flight_controller/airlink.md [hunk 11]
```diff
@@ -0,0 +1,1003 @@
...
+   <td>External UART</td>
+   </tr>
+   </tbody>
+   </table>
+
+
+
+## RC Input
+
+![RC Input](../../assets/flight_controller/airlink/airlink-rc-input.jpg)
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:891]: Image below text where possible.

File: en/flight_controller/airlink.md (hunk 2 above)
Comment (hamishwillee) [inline en/flight_controller/airlink.md:25]: All headings should be first-letter capitalized - just marking this one.

```suggestion
## Feature Highlights
```

File: en/flight_controller/airlink.md [hunk 12]
```diff
@@ -0,0 +1,1003 @@
...
+   - 3G HSPA+
+   - Approved by JRL/JTBL, FCC, PTCRB, RED, GCF
+   - Antenna, 4x4 MIMO
+   - Bands: Worldwide
+
+## Set content
+
+@[youtube](https://youtu.be/lex7axW8WQg)
+
+AIRLink set includes everything needed to setup the system and get prepared for the flight. Standard set contains:
```
Comment (hamishwillee) [inline en/flight_controller/airlink.md:85]:
```suggestion
AIRLink Enterprise arrives with everything needed to setup the autopilot system.

The standard set contains:
```

**Comment by hamishwillee** [comment]
@aviaks Looks great. Can you link the PRs where this went into PX4 and QGC?

I put some comments inline on the normal formatting rules. I can make some of these changes myself but I wanted you to be aware of what will happen.

Note, please also update en/SUMMARY.md to add this to the sidebar.

File: en/flight_controller/airlink.md (hunk 1 above)
Comment (aviaks) [reply en/flight_controller/airlink.md]: Yes, makes sense, removed now.

File: en/flight_controller/airlink.md (hunk 2 above)
Comment (aviaks) [reply en/flight_controller/airlink.md]: Thanks, done here and for all other places on the page.

File: en/flight_controller/airlink.md (hunk 3 above)
Comment (aviaks) [reply en/flight_controller/airlink.md]: Right, this is the description of the kit accessories. Both editions have the same kit accessories, the only difference is the enclosure of the unit. Now changed to "Kit Accessories".

File: en/flight_controller/airlink.md (hunk 8 above)
Comment (aviaks) [reply en/flight_controller/airlink.md]: Thanks! Moved higher on the page.

File: en/flight_controller/airlink.md (hunk 11 above)
Comment (aviaks) [reply en/flight_controller/airlink.md]: Done here and in a few other places where it was possible.

File: en/flight_controller/airlink.md (hunk 2 above)
Comment (aviaks) [reply en/flight_controller/airlink.md]: Thanks, done!

File: en/flight_controller/airlink.md (hunk 12 above)
Comment (aviaks) [reply en/flight_controller/airlink.md]: Changed, thanks!

**Comment by aviaks** [comment]
@hamishwillee Many thanks for reviewing and for the comments! Agree with all comments, committed, pushed and formatting rules noted for all future PRs.  

PX4 Firmware PR Link: https://github.com/PX4/PX4-Autopilot/pull/19529

en/SUMMARY.md updated as suggested.

File: en/flight_controller/airlink.md (hunk 9 above)
Comment (hamishwillee) [reply en/flight_controller/airlink.md:404]: Sounds great. No need to wait on it.

File: en/SUMMARY.md [hunk 13]
```diff
@@ -202,10 +202,11 @@
       * [Holybro pix32 (FMUv2)](flight_controller/holybro_pix32.md)
       * [mRobotics-X2.1 (FMUv2)](flight_controller/mro_x2.1.md)
       * [mRo Control Zero F7)](flight_controller/mro_control_zero_f7.md)
-      * [NXP RDDRONE-FMUK66 FMU](flight_controller/nxp_rddrone_fmuk66.md)
       * [ModalAI Flight Core v1](flight_controller/modalai_fc_v1.md)
       * [ModalAI VOXL Flight](flight_controller/modalai_voxl_flight.md)
       * [ModalAI VOXL 2](flight_controller/modalai_voxl_2.md)
+      * [NXP RDDRONE-FMUK66 FMU](flight_controller/nxp_rddrone_fmuk66.md)
+      * [Sky-Drones AIRLink](flight_controller/airlink.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:209]: FYI, all names are Manufacturer-FC name, alphabetical order

File: en/flight_controller/airlink.md (hunk 10 above)
Comment (hamishwillee) [reply en/flight_controller/airlink.md:368]: Great. If it was not in 1.13 we would have had to add a note.

**Comment by hamishwillee** [comment]
You're most welcome @aviaks - and thanks for making the markup changes. I made a few more - nothing to the meaning, cropped a few images, deleted the one that is no longer displayed, and fixed the position of the doc in the tree to match the order. 

FYI, it is in no way mandatory, but some manufacturers also choose to provide an assembly guide like this: https://docs.px4.io/main/en/assembly/quick_start_cuav_pixhawk_v6x.html
That can make things even easier for users.

Merging now.

File: en/SUMMARY.md (hunk 13 above)
Comment (aviaks) [reply en/SUMMARY.md:209]: Apologies and thanks for fixing!

**Comment by aviaks** [comment]
@hamishwillee thank you very much! 

Good point, we'll prepare and add assembly guide page. 

Peripheral pages PRs coming soon. 

Thanks again!

**Pull request merged**